include/lsst/afw/formatters/Utils.h

```diff
@@ -139,7 +141,24 @@ int extractVisitId(std::shared_ptr<lsst::daf::base::PropertySet const> const& pr
 int extractCcdId(std::shared_ptr<lsst::daf::base::PropertySet const> const& properties);
 int extractAmpId(std::shared_ptr<lsst::daf::base::PropertySet const> const& properties);
 
+/**
+Format a PropertySet into a single FITS header string
+
+@param[in] prop  Properties to format
+@return a FITS header string (exactly 80 characters per "card", no line terminators)
```

Comment:
To answer @PaulPrice's earlier question, yes. We unpersist several kinds of objects based on FITS headers (`Wcs`, `Calib`, `VisitInfo`) and it all works the same way: we read the FITS header into a `PropertyList`, then use the `PropertyList` to create whatever kind of object we want. This is not ideal, especially since `PropertyList` cannot handle all kinds of FITS headers (for instance those with a name but no value, meaning "unknown value"). However, it seems to have sufficed, so far.